etc/docker-compose/docker-compose.local.native.qs-rest-service.yml

```diff
@@ -0,0 +1,59 @@
+#version: "3.8"
+
+# run
+# docker compose -f <PROJECT_PATH>/etc/docker-compose/docker-compose.local.native.qs-rest-service.yml up --build --force-recreate
+#
+# run from project directory
+# docker compose -f ./etc/docker-compose/docker-compose.local.native.qs-rest-service.yml up --build --force-recreate
+#
+# enter container
+# docker exec -qs-rest-service-dev-native /bin/bash
+#
+# empty run
+# docker run --rm -it qs-rest-service:develop-local-native /bin/bash
+services:
+    qs-rest-service:
+        container_name: qs-rest-service-dev-native
+        build:
+            context: ../../
+#            dockerfile: etc/docker-compose/local.native-jar.Dockerfile
+            dockerfile: etc/docker-compose/local.native-micro-image.Dockerfile
+            args:
+                ICELL_JAVA_JRE_BASE_IMAGE: 'icellmobilsoft/base-java21jre:1.4.0'
```

Comment:
nativra is kell java image?

Reply:
2 verziot tettem be, az etc/docker-compose/local.native-jar.Dockerfile hasznalja ha nem a micro imaget választjuk ki végül, a sajátunkal is működik, igazából példának ott hagynám hogy ilyen image-ben is megy